kolibri/core/assets/src/vue/exercise-attempts/answer-icon.vue

```diff
@@ -28,13 +25,10 @@
     },
     computed: {
       isRight() {
-        return this.answertype === AnswerTypes.RIGHT;
-      },
-      isWrong() {
-        return this.answertype === AnswerTypes.WRONG;
+        return this.answertype.correct;
       },
       isHint() {
-        return this.answertype === AnswerTypes.HINT;
+        return this.answertype.hinted;
```

Comment:
I won't consider this to be coupling. The data entrance point is in the exercise-attempts/index.vue, we pass an specifically formatted array(the `log` props). It happens to be the same format as the masterlog's `pastattempts` field. And we discussed the advantages of this format. In fact the previous version of `pastattempts` field is of the same format this exercise-attempts use before.
Let me know if you want to chat about it.